NewsApp/NewsApp/View/de.lproj/Main.strings

```diff
@@ -52,3 +52,6 @@
 
 /* Class = "UILabel"; text = "Label"; ObjectID = "vKb-aX-5qg"; */
 "vKb-aX-5qg.text" = "Label";
+
+/* Class = "UISearchBar"; text = "Search for an article"; ObjectID = "mdt-Tt-gtQ"; */
+"mdt-Tt-gtQ.placeholder" = "Suchen einen Artikel";
```

Comment:
Est-ce que c'est des strings directement sur le storyboard ?

Reply:
fix in "Modification localization"